File: conditionals/import_time.py
# Malu Estevam, How to import time in Python
import time

# first instance of time in programming
# print(time.gmtime(0))

# current time in seconds since gmtime
current = time.time()

# current time as we are used to seeing it
now = time.ctime(current)
print(now)

# get just the hour
local_time = time.localtime(current)
hour = local_time.tm_hour
minutes = local_time.tm_min
print(hour)

File: conditionals/import_time.c
// Malu Estevam, How to import time
#include <stdio.h>
#include <time.h>



int main(void){
    // time since Jan 1 1970
    time_t seconds;
    
    // seconds = time(NULL);
    // printf("seconds since January 1, 1970 = %d\n", seconds);

    // current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);

    timeinfo = localtime(&rawtime);
    printf("Current time and date is %s\n", asctime(timeinfo));

    // current hour
    time_t now = time(NULL);
    struct tm *tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);

    return 0;
}